composer.json

```diff
@@ -86,7 +87,11 @@
         ]
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "composer/package-versions-deprecated": true,
+            "phpstan/extension-installer": true
+        }
```

Comment:
This is an important change and should be done in a separate PR to keep track of it in the history of Git

Reply:
This has no effect on consumers

Reply:
I know, but still we want to track it and not let it slip in the codebase as part of another PR

Reply:
Already done in master in 8fea3e3